rkyv/src/impls/arrayvec.rs

```diff
@@ -51,6 +51,24 @@ where
     }
 }
 
+impl<T, U, const CAP: usize> PartialEq<ArchivedVec<U>> for ArrayVec<T, CAP>
```

Comment:
I'm hesitant to provide generic `PartialEq` impls for generic external types after #434. See [this playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=aa694f15084ca15d1542bea1041e69ae) for an example. I'm fine with adding more `PartialEq` impls which are `for ArchivedType`s though.

Reply:
I'm curious about this actually wasn't there an `extra_trait` feature? https://github.com/rkyv/rkyv/pull/448 it's here but I can't find it anymore in 0.7 nor 0.8?

Reply:
There was, but I removed it because I wanted to have a more consistent API surface (not enable/disable the reverse impls with a feature). To compensate somewhat, there are new `get_with` methods implemented for all of the maps. I'd like to support your use case as best as I can, but also would prefer to avoid an `extra_traits` feature if possible.